### ISSUE TITLE
[Bug]: Tensorizer test is broken

### ISSUE THREAD

**Issue by alexeykondrat** [vllm-project/vllm#8249]
### Your current environment

Not applicable 

### 🐛 Describe the bug

@DarkLight1337 
Your PR https://github.com/vllm-project/vllm/pull/7836 broke Tensorizer test. 

Before: https://buildkite.com/vllm/ci-aws/builds/7580
After: https://buildkite.com/vllm/ci-aws/builds/7583

Will you be able to fix it?

### Before submitting a new issue...

- [X] Make sure you already searched for relevant issues, and asked the chatbot living at the bottom right corner of the [documentation page](https://docs.vllm.ai/en/latest/), which can answer lots of frequently asked questions.

**Comment by DarkLight1337** [comment]
Thanks for the heads up, I'm looking into it now.
